packages/taco/src/taco.ts

```diff
@@ -144,9 +137,8 @@ export const decrypt = async (
   provider: ethers.providers.Provider,
   domain: Domain,
   messageKit: ThresholdMessageKit,
-  authProvider?: EIP4361AuthProvider,
+  context?: ConditionContext,
   porterUri?: string,
```

Comment:
Note to self - implement suggestion here - https://github.com/nucypher/taco-web/pull/553#discussion_r1711397108 - before merging.